test/metabase/api/search_test.clj

```diff
@@ -370,6 +371,20 @@
       (is (= dashboard-count-results
              (set (unsorted-search-request-data :rasta :q "dashboard-count")))))))
 
+(deftest moderated-status-test
+  (let [search-term "moderated-status-test"]
+    (mt/with-temp [:model/Card {card-id :id} {:name "moderated-status-test"}]
+      (doseq [status ["verified" nil "verified"]]
```

Comment:
```suggestion
      (doseq [status ["verified" nil]]
```

Reply:
1. it needs to be verified at the end because an object can be unverified and still exist in `moderation_review`
2. I can make it verified once and test it, but I wanted to test cases where multiple moderation_review exists

p/s: I don't understand why we need to keep multiple moderation_review, maybe we can just keep one per item but that's the problem for another day.

Reply:
Got it. Can you add that to a `testing` description, it's not obvious by reading the test

Reply:
how about a comment? I don't think the setup matters much for the testing context here, as far as the test is concerned it's making sure the search endpoint returned the correct moderated status.

Reply:
Comment works too.